src/main/java/games/strategy/triplea/delegate/AirBattle.java

```diff
@@ -638,13 +638,8 @@ public void execute(final ExecutionStack stack, final IDelegateBridge bridge) {
   }
 
   static Match<Unit> attackingGroundSeaBattleEscorts(final PlayerID attacker, final GameData data) {
-    return new Match<Unit>() {
-      @Override
-      public boolean match(final Unit u) {
-        final Match<Unit> canIntercept = Match.all(Matches.unitCanAirBattle);
-        return canIntercept.match(u);
-      }
-    };
+    final Match<Unit> canIntercept = Matches.unitCanAirBattle;
+    return canIntercept;
```

Comment:
I left the intermediate variable here because it seems to provide some context and follows the same pattern in the subsequent two methods.  However, I did remove the `Match#all()` call because it doesn't offer anything when only a single component match is present.